robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java

```diff
@@ -27,8 +27,6 @@ public void testParseColor() throws Exception {
     assertThat(Color.parseColor("#00000000")).isEqualTo(0);
 
     assertThat(Color.parseColor("#ffaabbcc")).isEqualTo(-5588020);
-    assertThat(Color.parseColor("#fabc")).isEqualTo(-5588020);
-    assertThat(Color.parseColor("#abc")).isEqualTo(-5588020);
```

Comment:
Because we now actually test the Android Color.java (which doesn't actually support the #xxx format) we can't keep these. I left the test for regression.

 `ResourceHelper ` is what Android uses for processing resources at compile time so it makes sense for us to mirror that behavior, instead of hooking into `Color.parseColor()`.